src/exampleTypes/index.js

```diff
@@ -9,11 +9,10 @@ export Facet from './Facet'
 export FacetSelect from './FacetSelect'
 export Geo from './Geo'
 export Number from './Number'
-export PagedResultTable from './PagedResultTable'
 export Query from './Query'
 export ResultCount from './ResultCount'
 export ResultPager from './ResultPager'
-export ResultTable from './ResultTable'
+export ResultTable, { PagedResultTable } from './ResultTable'
```

Comment:
Better to deal with export the different names for the same component here instead of `ResultTable`